### ISSUE TITLE
Cannot build msi installer on v0.10.39

### ISSUE THREAD

**Issue by richardlau** [joyent/node#25563]
I'm no longer able to build msi installers on Windows after e7c84f82c730bddb7450dc56065aab11b93ffd30. 

````
...
Building node-0.10.39
Project file contains ToolsVersion="12.0". This toolset is unknown or missing. You may be able to resolve this by installing the appropriate .NET Framework for this toolset. Treating the project as if it had ToolsVersion="4.0".
Project file contains ToolsVersion="12.0". This toolset is unknown or missing. You may be able to resolve this by installing the appropriate .NET Framework for this toolset. Treating the project as if it had ToolsVersion="4.0".
C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.Cpp.Platform.targets(44,5): error MSB8020: The builds tools for v120 (Platform Toolset = 'v120') cannot be found. To build using the v120 build tools, either click the Project menu or right-click the solution, and then select "Update VC++ Projects...". Install v120 to build using the v120 build tools. [D:\build\jenkins\workspace\Node.js-Windows-Intel-32\node\tools\msvs\msi\custom_actions.vcxproj]
Project file contains ToolsVersion="12.0". This toolset is unknown or missing. You may be able to resolve this by installing the appropriate .NET Framework for this toolset. Treating the project as if it had ToolsVersion="4.0". 
````

My build environment is Visual Studio 2010 SP1 and WiX Toolset v3.6 on Windows 7, which is able to successfully build the msi installers for v0.10.38. I can also successfully build the installers on v0.10.39 if I revert e7c84f82c730bddb7450dc56065aab11b93ffd30.

**Comment by misterdjules** [comment]
@richardlau I had to fix a few issues (not identical to yours, but related) with Windows builds due to https://github.com/joyent/node/commit/e7c84f82c730bddb7450dc56065aab11b93ffd30, and came up with https://github.com/misterdjules/node/commit/5423d14482085887dfc3ac2d2ae2047943997a9d. It looks like it might fix your problem, would you mind giving this change a try and let me know if it fixes your problem?

**Comment by mhdawson** [comment]
Richard had tried the patches in that but had issues either while applying them or building after they were applied. I don't have the details but I'm sure he'll add when he gets in tomorrow.

**Comment by misterdjules** [comment]
Added as a P-2 as I don't think a significant number of users build their own installers. It would be nice to have that fixed before doing the next v0.10 release though, which is why I added it to the v0.10.40 milestone.

**Comment by misterdjules** [comment]
@richardlau https://github.com/misterdjules/node/commit/5423d14482085887dfc3ac2d2ae2047943997a9d is now available in [a pull request that I just created](https://github.com/joyent/node/pull/25569). Would you mind giving it a try and letting me know if this PR fixes your build problems?

Thank you!

**Comment by richardlau** [comment]
@misterdjules I applied the PR, which has fix the initial ToolsVersion problem, but now has the following error:

https://gist.github.com/richardlau/eae6275b90077b207753

I will try reinstalling WiX.

**Comment by misterdjules** [comment]
@richardlau Interesting, I haven't had that compilation error before, I'll try to reproduce it asap.

**Comment by richardlau** [comment]
I reinstalled WiX 3.6 but still get the error. I'll try updating WiX next.

**Comment by richardlau** [comment]
Tried WiX 3.6->3.8 and encountered the error. The error goes away and I can successfully build the installers with WiX 3.9 R2 (and #25569 ).

**Comment by misterdjules** [comment]
@richardlau Thanks for letting us know :+1:

**Comment by richardlau** [comment]
Verified fixed in v0.10 by #25569. :+1:
